### ISSUE TITLE
Notification for untranslated languages

### ISSUE THREAD

**Issue by Regnander** [sp614x/optifine#1403]
There are still a lot of languages that OptiFine has yet to be translated into, so why not add a notification to encourage the players to add one themselves if OptiFine doesn't have a `.lang` file for the selected language: 

>OptiFine doesn't have a translation in your language yet. If you are fluent in English, please click here to add one and help the community.

(And "click here" would be a link to the `lang` folder in this repo, just like at OptiFine's website.)

The most logical place for this notification would be in the language selection screen, but I doubt that it would be noticed there. Any suggestions?

**Comment by Regnander** [comment]
>I'd say to add a notification as a pop-up screen when changing Minecraft vanilla language, if there's no available OptiFine translation.

Good idea. And perhaps a pop-up on startup (that only appears once, of course) for those who doesn't tend to switch languages so often.

**Comment by Madis0** [comment]
Maybe Optifine should move their files to a translation platform first, like Transifex or Crowdin? I imagine there are some players eager to do this, but don't have the time-patience to do it all at once or learn how Git/Github works.

**Comment by DaDodger** [comment]
Yeah, @sp614x could just request an open source license on Crowdin for OptiFine repo, no charges and with advanced localisation features for free. All of us translators could go on with it and handle the translations in a proper plataform (with trusted proofreaders and stuff).

**Comment by Madis0** [comment]
But Optifine isn't open source, this repo only has documentation and languages...

**Comment by Regnander** [comment]
>Maybe Optifine should move their files to a translation platform first, like Crowdin, Transifex or OneSky?

What about the documentation? I think GitHub's folder view is easy enough for the average Joe to understand how it works, but is there a better/handier/easier way to present the documentation?

**Comment by Madis0** [comment]
Could use the [wiki pages](https://help.github.com/articles/about-github-wikis/) of GitHub, if that's better.
